apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-support-availability.php

```diff
@@ -88,6 +108,27 @@ public function get_all_support_eligibility() {
 		return rest_ensure_response( $response );
 	}
 
+	/**
+	 * Should return messaging eligibility
+	 *
+	 * @param \WP_REST_Request $request    The request sent to the API.
+	 *
+	 * @return WP_REST_Response
+	 */
+	public function get_messaging_support_eligibility( \WP_REST_Request $request ) {
+		$query_parameters = array(
+			'group'       => $request['group'],
+			'environment' => $request['environment'],
+		);
+		$body             = Client::wpcom_json_api_request_as_user( 'help/messaging/is-available?' . http_build_query( $query_parameters ) );
```

Comment:
```suggestion
		$body = Client::wpcom_json_api_request_as_user( '/help/messaging/is-available?' . http_build_query( $query_parameters ) );
```

I could be wrong but I think you need the starting slash here.

Reply:
I guess not, I can see an instance of it without. Forgive me : )

Reply:
I'd assume so too, but looking at the other calls in this file and some others, it seems both versions work? 🤔

Reply:
With a slash: https://github.com/Automattic/wp-calypso/blob/6f2928a02c45d0a52fcf96081ad13ec647431f05/apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-ticket.php#L78
Without: https://github.com/Automattic/wp-calypso/blob/6f2928a02c45d0a52fcf96081ad13ec647431f05/apps/editing-toolkit/editing-toolkit-plugin/help-center/class-wp-rest-help-center-authenticate.php#L44

Those are existing calls that I assume are working. We could definitely update them to be consistent (probably the starting slash is the best way to go).

Reply:
No worries, I appreciate the attention to detail! Those missing starting slashes in most cases _are_ problematic. So probably good to fix this in a follow-up patch.